script.rb

```diff
@@ -25,7 +26,15 @@
   file = agent.get(bundle_url)
   file.save
   analytics_file = File.read(file.filename)
-  session = GoogleDrive::Session.from_config("config.json")
+  credentials = Google::Auth::UserRefreshCredentials.new(
+    client_id: ENV['GOOGLE_CLIENT_ID'],
+    client_secret: ENV['GOOGLE_CLIENT_SECRET'],
+    scope: [
+      "https://www.googleapis.com/auth/drive",
+      "https://spreadsheets.google.com/feeds/",
+    ],
+    refresh_token: ENV['GOOGLE_REFRESH_TOKEN'])
+  session = GoogleDrive::Session.from_credentials(credentials)
```

Comment:
I adopt environment variable at Google Auth and process is successfully executed.

But I am not sure this is best.